Release v1.7/technische robuustheid (#450)
MinBZK/Algoritmekader: docs/voldoen-aan-wetten-en-regels/maatregelen/4-owk-08-feedbackloops.md

---
title: Bepaal welke feedbackloops van invloed zijn op het algoritme
id: urn:nl:ak:mtr:owk-08
toelichting: Stel vast op welke manier de uitkomst of de inzet van het algoritme van invloed kan zijn op het proces en de werking in een latere fase. Probeer deze ‘feedbackloops’ in kaart te brengen zodat ze mogelijk voorkomen kunnen worden of gemonitord kunnen worden op mogelijke negatieve effecten. 
levenscyclus:
- ontwerp
- ontwikkelen
onderwerp:
- technische-robuustheid-en-veiligheid

> - bias-en-non-discriminatie

Reviewer: hm, dit is volgens mij wel echt de link tussen robuustheid en bias. Vandaar mijn voorkeur om dit wel te laten staan.

rollen:
- projectleider
- ontwikkelaar
hide:
- navigation
- toc
---

<!-- tags -->

## Maatregel
Stel vast op welke manier de uitkomst of de inzet van het algoritme van invloed kan zijn op het proces en de werking in een latere fase. 
Probeer deze ‘feedbackloops’ in kaart te brengen zodat ze mogelijk voorkomen kunnen worden of gemonitord kunnen worden op mogelijke negatieve effecten. 

## Toelichting
Een feedbackloop kan zich voordoen wanneer de uitkomst van een algoritme wordt gebruikt als nieuwe input voor het algoritme. 
Deze feedbackloops kunnen een vertekend beeld van de werkelijkheid geven en de robuustheid van het algoritme over tijd in gevaar brengen. Dit is met name van belang wanneer algoritmes bijleren (continue of periodiek). 

!!! note "Opmerking"

    Merk op dat feedbackloops ook een positief effect kunnen hebben, wanneer bijvoorbeeld gebruikerservaring wordt meegenomen in de doorontwikkeling van het algoritme. 

> Er zijn verschillende vormen van feedbackloops, die worden uitgelegd aan de hand van een voorbeeld van een algoritme dat de hoogte voor een bepaalde subsidie bepaalt.

Reviewer: ```suggestion
Er zijn verschillende vormen van feedbackloops:
```

> - *sampling feedbackloop*: Wanneer de beslissing die volgt uit het algoritme effect heeft op de kans dat bepaalde groepen in een volgende selectie terechtkomen. In het voorbeeld van de uitkering betekent dit bijvoorbeeld dat een bepaalde groep geen nieuwe aanvraag meer doet door een eerdere beslissing van het algoritme. 

Reviewer: ```suggestion
- *sampling feedbackloop*: Wanneer de beslissing die volgt uit het algoritme effect heeft op de kans dat bepaalde groepen in een volgende selectie terechtkomen. 
```

- *individual feedbackloop*: Wanneer bijvoorbeeld de burger minder geld uit gaat geven aan boodschappen omdat deze weet dat het daarop beoordeeld wordt door het algoritme. Een ander voorbeeld is wanneer de mening of visie van een beoordelaar verandert door [het gebruiken van het algoritme](../../onderwerpen/bias-en-non-discriminatie.md#menselijke-bias)(het overnemen van de ‘vooroordelen van een systeem’). 

- *feature feedbackloop*: Bijvoorbeeld wanneer de uitkomst dat een subsidie niet verstrekt wordt, ook als kenmerk ‘eerdere weigering van subsidie’ wordt gebruikt door het algoritme. 

> - *outcome feedbackloop*: Bijvoorbeeld wanneer burgers die een subsidie toebedeeld worden (uitkomst) hierdoor meer geld gaan uitgeven, waardoor ze ook eerder in geldnood komen. Dit kan zelfs gebeuren wanneer ze misschien zonder de subsidie niet in geldnood waren gekomen.

Reviewer: ```suggestion
- *outcome feedbackloop*: Wanneer burgers of bedrijven op basis van de uitkomst ander gedrag gaan vertonen. In het voorbeeld van de subsidie kan dat bijvoorbeeld zijn dat burgers hun uitgavepatroon veranderen. 
```
